mantle/platform/qemu.go

```diff
@@ -107,13 +107,16 @@ func ParseDiskSpec(spec string) (*Disk, error) {
 	split := strings.Split(spec, ":")
 	var size string
 	multipathed := false
+	sectorSize := 0
```

Comment:
What about defaulting it at 512?

Reply:
When `SectorSize` is 0, we just let the default QEMU behaviour kick in, which is 512 logical/physical sector sizes.

Reply:
(In Rust this would be an `Option<u32>` which would better express semantics)

Reply:
Ah, I see that makes sense! cool thank you.